boefjes/boefjes/plugins/kat_dns/schema.json

```diff
@@ -10,10 +10,11 @@
     },
     "REMOTE_NS": {
       "title": "REMOTE_NS",
-      "maxLength": 45,
+      "maxLength": 15,
       "type": "string",
       "description": "The IP address of the DNS resolver you want to use.",
       "default": "1.1.1.1"
     }
-  }
+  },
+  "required": []
```

Comment:
This stanza should not be needed, did you see any errors that indicated it was in fact needed?

Reply:
Indeed, it is not needed, I wanted to make it conform to other Boefjes like:
boefjes/boefjes/plugins/kat_nmap_ip_range/schema.json
boefjes/boefjes/plugins/kat_dns/schema.json
boefjes/boefjes/plugins/kat_nmap_udp/schema.json
boefjes/boefjes/plugins/kat_masscan/schema.json
Perhaps they do not need it either?

Reply:
Addressed in 7020509b021935dcf87f20cc0fab976f4f69a37e.